src/backend/SyncBackendConnection.cpp

```diff
@@ -655,6 +660,39 @@ std::vector<StatisticsData> SyncBackendConnection::get_data(
     }
 }
 
+std::vector<StatisticsData> SyncBackendConnection::change_unit_magnitude(
+        std::vector<StatisticsData>& data,
+        DataKind data_kind)
+{
+    switch (data_kind)
+    {
+        case DataKind::FASTDDS_LATENCY:
+        case DataKind::NETWORK_LATENCY:
+
+            // Convert from ns to ns
+            for (StatisticsData& point : data)
+            {
+                point.second /= 1000.0;
+            }
+            break;
+
+        case DataKind::PUBLICATION_THROUGHPUT:
+        case DataKind::SUBSCRIPTION_THROUGHPUT:
+
+            // Convert from b/s to kb/s
+            for (StatisticsData& point : data)
+            {
+                point.second /= 1024.0;
```

Comment:
If we are using Kilobytes, then the division is by 1000. However, if we'd have wanted to use Kibibytes (KiB), the the division is by 1024 as you stated here.
```suggestion
                point.second /= 1000.0;
```